README.md

```diff
@@ -234,11 +229,94 @@ At this point, the `Podman` docs claim that one should have moved over to `Podma
 1. In `$HOME/.docker/config.json` replace "credsStore" with "credStore" (minus an 's') to solve registry credentialing problems.
 2. Set `export DOCKER_HOST=<your_podman_machine_socket_path>` to the `Podman machine`'s socket on your system, which you can find in the `Resources` tab of `Podman Desktop`. The path starts with `unix://`.
 
-### Build and Run
+Podman is a daemonless container engine that's compatible with Docker containers. Here's how to set it up:
 
-8Knot is a multi-container application.
+1. **If transitioning from Docker Desktop:**
+   - Uninstall Docker Desktop (requires GUI uninstall and checking for remnants)
+   - Remove any remaining Docker files/configurations
 
-The app-server, worker-pools, redis-cache, and postgres-cache containers communicate with one another via docker networking.
+2. **Install Podman:**
+   - On macOS: `brew install podman podman-desktop`
+   - On Linux: Follow the [official installation guide](https://podman.io/getting-started/installation)
+
+3. **Initial Setup:**
+   - On macOS: Initialize and start a Podman machine:
+     ```bash
+     podman machine init
+     podman machine start
+     ```
+   - Install docker-compose (used by Podman Compose):
+     ```bash
+     brew install docker-compose  # on macOS
+     # or use your system's package manager
+     ```
+
+4. **Verify Installation:**
+   ```bash
+   podman --version
+   podman compose --version
+   ```
+
+5. **Optional but recommended to install Podman Desktop** 
+
+Further information below:
+
+#### Using Podman
+
+If you're using Podman, you can use either `podman compose` or `docker compose` (thanks to the compatibility layer). Here are both options:
+
+Using podman-compose directly:
+```bash
+# Start the application
+podman compose up --build
+
+# Scale worker pools (recommended)
+podman compose up --build --scale worker-query=2 --scale worker-callback=2
+
+# Stop the application
+ctrl-c
+
+# Clean up containers and volumes
+podman compose down --volumes
+```
+
+Using docker compose with Podman:
+```bash
+# The same Docker Compose commands work with Podman if you've set up the compatibility layer
+docker compose up --build
+docker compose up --build --scale worker-query=2 --scale worker-callback=2
+```
+
+Additional Podman-specific commands that might be helpful:
+```bash
+# List all containers
+podman ps -a
+
+# View container logs
+podman logs <container_name>
+
+# Remove all containers and pods
+podman pod rm -f -a
+
+# Clean up unused images and volumes
+podman system prune --volumes
+```
```

Comment:
This is where the information about transitioning between docker to podman should be. Section title something like "Further information on transitioning from Docker desktop to Podman"